### ISSUE TITLE
Experimental support for decorators is a feature that is subject to change in a future release. Set the 'experimentalDecorators' option to remove this warning.

### ISSUE THREAD

**Issue by trusktr** [TypeStrong/atom-typescript#1453]
After replacing `ide-typescript` with `atom-typescript`, typing seems to work better in some cases. 👍 

However I see this:

```
Experimental support for decorators is a feature that is subject to change in a future release. Set the 'experimentalDecorators' option to remove this warning.
```

and I do have

```js
    "experimentalDecorators": true,
```

in my `tsconfig.json`. Any idea why it is complaining then?

**Comment by lierdakil** [comment]
Can't repro with typescript-3.0.1. Make sure `experimentalDecorators` is set in `compilerOptions` and not on the top level (at least that's what I was bitten by more than once). Also, make sure that atom-typescript is picking up the correct `tsconfig.json` (which can be fiddly sometimes in a multi-tsconfig project) -- you can open `tsconfig.json` that TypeScript thinks is related to the currently-open file by clicking on `.` or `~dirnname` in the status bar:
![image](https://user-images.githubusercontent.com/7275622/46735365-b3e79d80-cc9e-11e8-87e7-2e52482a446b.png)
e.g. here, that would be `~lib`

**Comment by trusktr** [comment]
Ah, interesting, in the project in question, I do have a single `tsconfig.json` at the root of the project, but I see this in the Atom status bar:

<img width="210" alt="screen shot 2018-10-29 at 6 44 52 pm" src="https://user-images.githubusercontent.com/297678/47690308-d4728b80-dbaa-11e8-8480-59c660a4603a.png">

**Comment by trusktr** [comment]
Is that because atom-typescript expects ts v3?

**Comment by lierdakil** [comment]
No, it's not because a particular TypeScript version is expected.

Basically, config resolver failed to find an appropriate tsconfig.json. This might be a bug in the config resolver, but more often than not, such stuff is caused by configuration errors. At the moment I'm thinking that the file you were viewing when you took that screenshot probably either fell under the `exclude` glob in tsconfig, or if you're using `files` directive, it's likely missing in `files`.

Either way, if you can share your `tsconfig.json` and the project directory tree (i.e. if you are legally permitted to do so), please do, this might make pinpointing the issue easier.
